js/src/layers/GeoJSON.js

```diff
@@ -16,9 +16,9 @@ var LeafletGeoJSONModel = featuregroup.LeafletFeatureGroupModel.extend({
 var LeafletGeoJSONView = featuregroup.LeafletFeatureGroupView.extend({
     create_obj: function () {
         var that = this;
-        var model_style = this.model.get('style');
         style = function (feature) {
-            return _.extend(feature.properties.style, model_style);
+            var model_style = that.model.get('style');
+            return _.extend(feature.properties.style || {}, model_style);
```

Comment:
Awesome, thanks for that!

Reply:
Sorry, about multiple changes in one PR, I'll rebase this one to just GeoJSON, and do separate one for global `L`, and will leave out resize changes all together.

Reply:
No, don't bother wasting time for that. It was just a comment for next time :)